CHANGES.rst

```diff
@@ -4,6 +4,9 @@ Changelog
 1.0b2 (unreleased)
 ------------------
 
+- #19 repeat over the output of the log_tail method to have the audit log line-by-line.
```

Comment:
I think this line is not adding information significant enough for the changelog so I would prefer to remove it.